casepro/utils/__init__.py

```diff
@@ -164,3 +168,23 @@ def uuid_to_int(uuid):
     "Values from -2147483648 to 2147483647 are safe in all databases supported by Django"
     """
     return UUID(hex=uuid).int % (2147483647 + 1)
+
+
+def get_language_name(iso_code):
+    """
+    Gets a language name for a given ISO639-2 code.
```

Comment:
RapidPro uses ISO639-2 for language - if other deployments use a different code set then maybe do this translation in the backend abstraction.